runtime/gc_base/ObjectAccessBarrier.hpp

```diff
@@ -541,6 +541,17 @@ class MM_ObjectAccessBarrier : public MM_BaseVirtual
 		/* no-op default implementation */
 	}
 
+	virtual J9Object* referenceGet(J9VMThread *vmThread, J9Object *refObject)
```

Comment:
I'd like to see consistent formatting (e.g. `J9Object *` instead of `J9Object*`).